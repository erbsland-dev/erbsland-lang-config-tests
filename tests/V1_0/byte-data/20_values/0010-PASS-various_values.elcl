# Erbsland Configuration Language Test File
[byte_data_values]
value 01: <>
value 02: < >
value 03: <	>
value 04: <01>
value 05: <af>
value 06: <AF>
value 07: < 23 >
value 08: <	45	>
value 09: <67 89>
value 10: < ab cd >
value 11: <	ef	AB	>
value 12: <    	 CD    	   EF   	  >
value 13: <0123 4567 89ab cdef 0123 4567 89ab cdef 0123 4567 89ab cdef 0123 4567 89ab cdef 0123 4567 89ab cdef>
value 14: <hex:>
value 15: <hex: >
value 16: <hex:	>
value 17: <hex:01>
value 18: <hex:af>
value 19: <hex:AF>
value 20: <hex: 23 >
value 21: <hex:	45	>
value 22: <hex:67 89>
value 23: <hex: ab cd >
value 24: <hex:	ef	AB	>
value 25: <hex:    	 CD    	   EF   	  >
value 26: <hex:0123 4567 89ab cdef 0123 4567 89ab cdef 0123 4567 89ab cdef 0123 4567 89ab cdef 0123 4567 89ab cdef>
