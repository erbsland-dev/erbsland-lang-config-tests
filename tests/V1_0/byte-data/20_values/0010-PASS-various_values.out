byte_data_values = SectionWithNames()
byte_data_values.value_01 = Bytes()
byte_data_values.value_02 = Bytes()
byte_data_values.value_03 = Bytes()
byte_data_values.value_04 = Bytes(01)
byte_data_values.value_05 = Bytes(af)
byte_data_values.value_06 = Bytes(af)
byte_data_values.value_07 = Bytes(23)
byte_data_values.value_08 = Bytes(45)
byte_data_values.value_09 = Bytes(6789)
byte_data_values.value_10 = Bytes(abcd)
byte_data_values.value_11 = Bytes(efab)
byte_data_values.value_12 = Bytes(cdef)
byte_data_values.value_13 = Bytes(0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef)
byte_data_values.value_14 = Bytes()
byte_data_values.value_15 = Bytes()
byte_data_values.value_16 = Bytes()
byte_data_values.value_17 = Bytes(01)
byte_data_values.value_18 = Bytes(af)
byte_data_values.value_19 = Bytes(af)
byte_data_values.value_20 = Bytes(23)
byte_data_values.value_21 = Bytes(45)
byte_data_values.value_22 = Bytes(6789)
byte_data_values.value_23 = Bytes(abcd)
byte_data_values.value_24 = Bytes(efab)
byte_data_values.value_25 = Bytes(cdef)
byte_data_values.value_26 = Bytes(0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef)
